quickstart_guides/rails.md

```diff
@@ -0,0 +1,85 @@
+address: support.aptible.com/quickstart/rails
+
+### Ruby on Rails Quickstart
+
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+Use the `apps:create` command: `aptible apps:create $HANDLE`
```

Comment:
Maybe change this to `$APP_HANDLE` (and the `$HANDLE` referenced for database below to `$DB_HANDLE` to avoid confusion).